Source/Scene/ModelExperimental/FeatureIdPipelineStage.js

```diff
@@ -318,6 +318,13 @@ function processImplicitRange(
   );
 }
 
+const shiftsByChannelCount = [
```

Comment:
I'm wondering if this is best done as a builtin function, `czm_unpackUint()` overloaded for all 4 types, right now this makes for a messy one-liner like this:

```
" featureIds.featureId_0 = dot(floor(texture2D(u_featureIdTexture_0, v_texCoord_0).rgba * 255.0 + 0.5), vec4(1.0, 256.0, 65536.0, 16777216.0));"
```

Reply:
Talking with @lilleyse, we decided to create such a function, 

`int czm_unpackUint(genType vec)` -- and just explain the limitations.

Reply:
Also note that the `floor(x * 255 + 0.5)` should be a part of this function.